sass/civicrm-override/_calender.scss

```diff
@@ -0,0 +1,194 @@
+.ui-datepicker {
+  background: $crm-white;
+  border: 1px solid $datepicker-border-color;
+  border-radius: 0;
+  box-shadow: $box-shadow;
+  min-width: 160px;
+  padding: 10px;
+  width: auto;
+
+  &,
+  * {
+    border-radius: 0;
+    font-family: $font-family-base;
+  }
+
+  * {
+    background: none;
+    box-shadow: none;
+  }
+
+  &,
+  .ui-datepicker-calendar {
+    td[data-handler] a {
+      background: $gray-lighter;
+    }
+
+    td[data-handler] a.ui-state-highlight {
+      border-color: $brand-primary;
+    }
+  }
+}
+
+.ui-datepicker .ui-datepicker-header {
+  border: 0;
+  margin-bottom: 5px;
+
+  .ui-datepicker-next,
+  .ui-datepicker-prev {
+    box-shadow: none;
+    height: 28px;
+    top: 4px;
+
+    &::before {
+      display: inline-block;
+      line-height: 30px;
+    }
+  }
+
+  .ui-datepicker-prev::before {
+    @include fa-icon(13px, '\f0d9');
+    padding-left: 5px;
+  }
+
+  .ui-datepicker-next::before {
+    @include fa-icon(13px, '\f0da');
```

Comment:
I don't think we will reuse those icon in this extension any more.  but i'll replaced 13px with variable $font-size-base (plz check my new commit)
https://github.com/compucorp/uk.co.vedaconsulting.mosaico/pull/13/commits/14d325f0f2b8aa34418a6421b4991e4c1931437d